enterprise/backend/test/metabase_enterprise/sandbox/api/user_test.clj

```diff
@@ -19,4 +22,20 @@
                :first_name  "Rasta"
                :email       "rasta@metabase.com"
                :id          true}]
-             (tu/boolean-ids-and-timestamps ((mt/user-http-request :rasta :get 200 "user") :data)))))))
+             (tu/boolean-ids-and-timestamps ((mt/user-http-request :rasta :get 200 "user") :data))))
+      (testing "When a group manager"
```

Comment:
I think we could be more descriptive here.
Sth like: `Should returns self if group manager is sandboxed (#23246)`